salesagility/SuiteCRM
# Fix multiple notification emails on Lead Creation WF

Prevents a bug where workflows cause multiple Lead assignment notifications to be sent.

## Description



Fairly rare bug where each of the resaves of a workflow cause another email notification to be sent.

## How To Test This


1. Ensure outgoing emails are working
2. Ensure "Assignment Notifications" is enabled in Email Settings
3. Have another user created other than Admin, make sure this user has an email address in their profile
4. Create a Workflow to run on new Lead Records. 
5. Set no conditions on this WF. 
6. Set at least one "modify record" action on this WF, doesn't matter which, for example field title = foo.
7. As Admin user, create a new Lead, make sure you set assigned user to the one you created earlier.
8. See the user has received 2 emails notifying of a new Lead instead of only one.

## Types of changes


### Final checklist
